### ISSUE TITLE
Initial Build of Project

### ISSUE THREAD

**Issue by deidyomega** [CodeForFoco/codeforfoco.github.io#102]
Following guide in readme.md I ran the final command

    bundle exec jekyll serve

and the error I got was:

    Liquid Exception: no implicit conversion of Fixnum into String in /_layouts/default.html

I checked default.html, and there is no number => string conversion, thus it must be in a sub file.  As I've never used jekyll, I could use some help on this one.

**Comment by abettermap** [comment]
I'm not very familiar with Jekyll myself but someone else had an [identical error](https://github.com/Shopify/liquid/issues/904).

Which versions of Jekyll and Liquid do you have?

```
gem list | grep jekyll
gem list | grep liquid
```

I had `3.3.1` and `3.0.6`, respectively, without errors, and it is also working for me using Jekyll `3.4.4`. Try this and see if it fixes it for you:

```
git fetch origin && git checkout 31004c4
bundle update jekyll
bundle exec jekyll serve
```

**Comment by deidyomega** [comment]
Hmm, that works!

`gem list | grep jekyll`
* jekyll (3.5.0)
* jekyll-autoprefixer (1.0.1)
* jekyll-feed (0.9.2)
* jekyll-gist (1.4.1)
* jekyll-mentions (1.2.0)
* jekyll-paginate (1.1.0)
* jekyll-sass-converter (1.5.0)
* jekyll-sitemap (1.1.1)
* jekyll-watch (1.5.0)

`gem list | grep liquid`
* liquid (4.0.0)

**Comment by deidyomega** [comment]
So on a final note:

I'd either recommend creating a PR for whatever magic you did to make this work *OR* updating the gemfile to this:

`gem 'jekyll', ">=3.0", "<=3.4.4"`

Since the problem seems to come from too high a jekyll (https://github.com/Shopify/liquid/issues/904)

**Comment by JordanMajd** [comment]
Fixed with #108
